uvicorn/config.py

```diff
@@ -202,7 +202,7 @@ def __init__(
 
     @property
     def asgi_version(self) -> str:
-        return {"asgi2": "2.0", "asgi3": "3.0"}[self.interface]
+        return {"asgi2": "2.0", "asgi3": "3.0", "wsgi": "2.0"}[self.interface]
```

Comment:
The `WSGIMiddleware` uses the ASGI3 callable interface, so I _think_ we should use `"wsgi": "3.0"`.

Reply:
done!